src/libs/actions/PaymentMethods.js

```diff
@@ -28,4 +32,32 @@ function getPaymentMethods() {
         });
 }
 
-export default getPaymentMethods;
+/**
+ * Calls the API to add a new card.
+ *
+ * @param {Object} params
+ */
+function addBillingCard(params) {
+    API.AddBillingCard({
+        cardNumber: params.cardNumber,
+        cardYear: params.expirationDate.substr(3),
+        cardMonth: params.expirationDate.substr(0, 2),
+        cardCVV: params.securityCode,
+        addressName: params.nameOnCard,
+        addressZip: params.zipCode,
+        currency: CONST.CURRENCY.USD,
+    }).then(((response) => {
+        if (response.jsonCode === 200) {
+            Onyx.set(ONYXKEYS.CARD_LIST, response);
```

Comment:
So right here we're overwriting the entire `ONYXKEYS.CARD_LIST` key with the response from the call which is just this:
```
{
fundID: 21
jsonCode: 200
requestID: "2icDJ6"
}
```

What we actually want to do is use the information we were originally given to add a new card to the list.

The format the cards are stored in is:
```
{
  "additionalData": {
    "isBillingCard": true,
    "isP2PDebitCard": false
  },
  "addressName": "Asdf Asdf", //User's name on the card
  "addressState": "US",
  "addressStreet": "",
  "addressZip": 80026,
  "cardMonth": 9,
  "cardNumber": "520082XXXXXX8210",
  "cardYear": 2023,
  "created": "2021-09-16 00:42:18",
  "currency": "USD",
  "fundID": 21, //This is returned from the api as mentioned above
}
```

So when we get a response that is 200, we should do an `Onyx.merge(<card object in the format above>)`

Reply:
The only info we don't have is the `created` which is fine, we shouldn't need that.

Reply:
We also need to mask the card number they input to match the format above